Mathlib/Computability/Language.lean

```diff
@@ -297,3 +297,12 @@ instance : KleeneAlgebra (Language α) :=
       exact le_trans (le_mul_congr h le_rfl) ih }
 
 end Language
+
+/-- Symbols for use by all kinds of grammars. -/
+inductive Symbol (T : Type*) (N : Type*)
+  /-- Terminal symbols (of the same type as the language) -/
+  | terminal    (t : T) : Symbol T N
+  /-- Nonterminal symbols (must not be present at the end of word being generated) -/
+  | nonterminal (n : N) : Symbol T N
+  -- If we have `[DecidableEq T]` and `[DecidableEq N]` then `[DecidableEq (Symbol T N)]` is added.
+  deriving DecidableEq
```

Comment:
I think the comment is unnecessary, and maybe `deriving Fintype` would be nice too?

Reply:
`Repr` would be another good one.

Reply:
default handlers have not been implemented yet, class: 'Fintype' types: [Symbol]

Reply:
I think you're probably just missing an import

Reply:
I pushed the fix